.travis.yml

```diff
@@ -11,3 +11,17 @@ script:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+
+deploy:
+  # deploy release
+  - provider: script
+    script: ./gradlew bintrayUpload install -PdryRun=false
```

Comment:
I prefer to use a `deploy_travis.sh` or similar rather than the script here directly